package.json

```diff
@@ -22,6 +22,7 @@
     "validation"
   ],
   "devDependencies": {
+    "camelcase-keys": "4.0.0",
```

Comment:
Let's remove this dependency (I'm allergic to extra deps :) and just write a simple transformer test instead